### PR TITLE
Ports: vitetris

### PR DESCRIPTION
**Pull request by JaanVr**
A quick port of vitetris. Only one small hack had to be done to get it to compile and run.

### CODE REVIEW

File: Ports/vitetris/package.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+useconfigure="true"
+version="0.59.1"
+files="https://github.com/vicgeralds/vitetris/archive/refs/tags/v${version}.tar.gz vitetris.tar.gz"
+configopts="--without-xlib --without-joystick --without-network"
+
+configure() 
+{
+    chmod +x "${workdir}"/"$configscript"
+    run ./"$configscript" $configopts
+ }
```
Comment (linusg) [inline Ports/vitetris/package.sh:12]:
```suggestion
configure() {
    run chmod +x "$configscript"
    run ./"$configscript" $configopts
 }
```

I realise this is taken from `.port_include.sh`, but using `run` in both cases should be preferred.

**Review by linusg — CHANGES_REQUESTED** [review]

**Review by linusg — COMMENTED** [review]
Please squash both commits as per the contributing guidelines: https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md

File: Ports/vitetris/package.sh (hunk 1 above)
Comment (JaanVr) [reply Ports/vitetris/package.sh]: Fixed

**Comment by JaanVr** [comment]
> Please squash both commits as per the contributing guidelines: https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md

Done

**Pull request merged**